core/domain/user/crud.py

```diff
@@ -0,0 +1,68 @@
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import models
+from core.domain.user import schemas
+from core.domain.user.exceptions import EmailAlreadyExistsError
+
+
+async def get_users(db: AsyncSession, skip: int = 0, limit: int = 10):
```

Comment:
Бест практис - проставлять возвращаемый тип данных. Этот круд возвращает списое моделей алхимии, т.е.
 ```suggestion
async def get_users(db: AsyncSession, skip: int = 0, limit: int = 10) -> list[User]:
```